src/test/java/org/opensearch/flowframework/workflow/CreateConnectorStepTests.java

```diff
@@ -59,30 +64,17 @@ public void setUp() throws Exception {
                 Map.entry("protocol", "test"),
                 Map.entry("params", params),
```

Comment:
Note to reviewers: this is missing test coverage because the key "params" here doesn't match the `PARAMETERS_FIELD` String value, implied by the replaced strings in the ConnectorAction request body.  Should this be "parameters"?  If so I'll fix it.